dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/model/WorkflowHistoryTest.java

```diff
@@ -0,0 +1,51 @@
+package com.dotmarketing.portlets.workflows.model;
+
+import com.liferay.util.StringPool;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Map;
+
+public class WorkflowHistoryTest {
+
+    @Test
+    public void test_getChangeMap_null_description ()  {
+
+        final WorkflowHistory workflowHistory = new WorkflowHistory();
+        workflowHistory.setChangeDescription(null);
+        final Map<String, Object> changeMap = workflowHistory.getChangeMap();
+
+        Assert.assertNotNull("GetChangeMap must not return a null map",changeMap);
+        Assert.assertEquals("Should return a blank description", changeMap.get("description"), StringPool.BLANK);
+        Assert.assertEquals("Should return a comment type", changeMap.get("type"), WorkflowHistoryType.COMMENT.name());
+        Assert.assertEquals("Should return a comment type", changeMap.get("state"), WorkflowHistoryState.NONE.name());
+    }
+
+    @Test
+    public void test_getChangeMap_with_description ()  {
+
+        final WorkflowHistory workflowHistory = new WorkflowHistory();
+        workflowHistory.setChangeDescription("test");
+        final Map<String, Object> changeMap = workflowHistory.getChangeMap();
+
+        Assert.assertNotNull("GetChangeMap must not return a null map",changeMap);
+        Assert.assertEquals("Should return test description", workflowHistory.getChangeDescription(), "test");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Should return test description" appears 4 times in this file; the first occurrence is on line 31](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4904110)